crates/bindings-csharp/BSATN.Codegen/BSATN.Codegen.csproj

```diff
@@ -31,13 +33,6 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.3.1" PrivateAssets="all" />
   </ItemGroup>
 
-  <ItemGroup>
-    <None Include="../logo.png" Pack="true" PackagePath="" />
-    <None Include="../Codegen/README.md" Pack="true" PackagePath="" />
-    <None Include="../LICENSE" Pack="true" PackagePath="" />
-    <None Include="$(OutputPath)/$(AssemblyName).dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
-  </ItemGroup>
```

Comment:
Why did we get to remove these?

Reply:
Because BSATN.Codegen is no longer packable, and all of these are packing directives for extra files.

Reply:
Ah yeah I see now they're still included in `Runtime.csproj`